node/components.go

```diff
@@ -44,6 +45,7 @@ func bridgeComponents(cfg *Config, store Store) fx.Option {
 		nodecore.Components(cfg.Core),
 		fx.Supply(header.MakeExtendedHeader),
 		fx.Provide(services.LightAvailability), // TODO(@Wondertan): Remove strict requirements to have Availability
```

Comment:
You can remove this TODO now as it is irrelevant and make it Full, as discussed in sync.